hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/FileTestBase.java

```diff
@@ -276,11 +277,13 @@ protected void givenValidFile(boolean deleted, boolean withKeys) {
         fileWithNoContent = new File(fileId, expirationTime, withKeys ? keys : null, null, memo, deleted);
         fileSystem =
                 new File(fileSystemFileId, expirationTime, withKeys ? keys : null, Bytes.wrap(contents), memo, deleted);
+        upgradeFile = new File(
```

Comment:
This is using the upgrade file a bit too consistently, which invalidates some of the testing.
The upgrade file probably shouldn't be set here.

Reply:
removed thanks